mkosi/resources/mkosi.md

```diff
@@ -1363,26 +1364,33 @@ Scripts executed by mkosi receive the following environment variables:
   [SOURCE_DATE_EPOCH](https://reproducible-builds.org/specs/source-date-epoch/)
   for more information.
 
+* `$MKOSI_UID` is the uid of the user that invoked mkosi, potentially
+  translated to a uid in the user namespace that mkosi is running in.
+
+* `$MKOSI_GID` is the gid of the user that invoked mkosi, potentially
+  translated to a gid in the user namespace that mkosi is running in.
```

Comment:
This should note that it can be used to run things as the owning user, to e.g. avoid git complaining.